### DISCUSSION TITLE
Are these being produced?

### DISCUSSION THREAD

**Discussion by ohhai-kthxbai** [pilotak/eth-io#2] [Q&A] [answered]
Hi. Are you making/selling these?

**Comment by pilotak** [comment, marked as answer]
Hi,
I would like to but it’s very difficult to get the some parts this year due to a global shortage and unfortunately i’m run out of my personal stock. Of course I can still buy them but from several suppliers resulting in multiple P&P which would increase the total cost. Not even talking about shipping time…

Anyway this is not a commercial product, it's a personal one for everyone to make. That's why i released it as open source.

**Reply by selerzynski** [reply]
Hello! I'm interested with having for tests this setup - i would like to order an assembly for this - question is: can u please shere also BOM file and CPL file ? THX

**Comment by green2blue** [comment]
Hi, i would realy buy one of your boards, maybe as a kit for self assembly. How can i get my hands on one of your boards? This is a perfekt fit for ESPhome. Thank you very much! 
I am from Austria and would realy appreceate if you would sell me one ore more...
Could you give me your mail adress so i can write you directly?
Kind regards

**Reply by pilotak** [reply]
Hi, nothing changed since May when I last posted the answer. I obviously realize that this would be a nice product to sell although the price would be bit steep but this is due to many features packed into one board - not everyone would need them all.

You can of course find my email address in the commits history should you wish, i won't post it here.

**Reply by green2blue** [reply]
Hi, i am not able to find your address, maybe i am not smart enought ;-) 
Is there a way to write you a pm?
Do you thought about to edit your desing and use a board that already has ethernet on board, because they are easyer to get.
Do you know this board https://www.olimex.com/Products/IoT/ESP32/ESP32-GATEWAY/open-source-hardware ?

**Reply by pilotak** [reply]
here is a tutorial for the email address extraction https://www.nymeria.io/blog/how-to-manually-find-email-addresses-for-github-users it works!

I don't quite get why would i edit the PCB for Olimex board, i used modules too (one for ESP32 and one for ethernet, both widely available on Aliexpress). Of course you can get https://www.cnx-software.com/2020/11/09/6-wt32-eth01-is-a-tiny-esp32-board-with-ethernet/ but in case it gets broke, it would be hard to desolder. The problem is in the rest of the parts...

**Comment by selerzynski** [comment]
Hello! I'm interested with having for tests this setup - i would like to order an assembly for this - question is: can u please shere also BOM file and CPL file ? THX

**Comment by pilotak** [comment]
I don't know what the CPL file is but BOM is already in the [wiki](https://github.com/pilotak/eth-io/wiki/BOM-(Bill-of-material))

**Comment by selerzynski** [comment]
Hello!

THX for answer - Question - are u producing this assebleys.
I'm really exitedwith this project - it is covering 90% of my needs for the
near future....
I would be interested in 3x main board and 2 times extnetion board.

BR
Piotr Selerzyński

niedz., 5 gru 2021 o 14:02 Pavel S ***@***.***> napisał(a):

> I don't know what the CPL file is but BOM is already in the wiki
> <https://github.com/pilotak/eth-io/wiki/BOM-(Bill-of-material)>
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/pilotak/eth-io/discussions/2#discussioncomment-1752223>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVCACLGBLTZQW5O7S4BEO63UPNPGNANCNFSM45ZVYTOA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
*Pozdrawiam *

*Selerzyński Piotr *

**Reply by ugomark** [reply]
Hi!
I'm interested too in getting this fantastic assemblies.
Unfortunately donato have skills to build it by myself.

@selerzynski did you find an assembler that u can share?

Many thanks

@pilotak tank you for sharing this project

**Comment by pilotak** [comment]
I would like to make them, but unfotunatelly parts are out of stock and it's not going to change in near future due to a global chip shortage. You can of course redesign the board to fit your components if you have them, Eagle files are uploaded or maybe use some chinese stock.

**Comment by kamyk** [comment]
Is it possible to order at least a board for that? I will collect and check mine storage to used parts.

**Comment by pilotak** [comment]
@kamyk Yes but i would need to order them from China which will cost the P&P + P&P to your country so it would make more sense to order it yourself and you will pay only one P&P

**Reply by kamyk** [reply]
> @kamyk Yes but i would need to order them from China which will cost the P&P + P&P to your country so it would make more sense to order it yourself and you will pay only one P&P

Could you share with me a board schematic or files? I'm happy to order it by myself. TBH - I tried to found it but no luck.

**Reply by pilotak** [reply]
it's there https://github.com/pilotak/eth-io/tree/master/hw including gerbers...

**Reply by kamyk** [reply]
Thank you! I will check and maybe order some.

**Comment by kamyk** [comment]
One more question: do you have somewhere in zip list of elements for JLCPCB to install SMD's?

**Reply by pilotak** [reply]
No, i solder them by hand at home. Overall BOM is [here](https://github.com/pilotak/eth-io/wiki/BOM-(Bill-of-material))
